src/components/Header.js

```diff
@@ -1,6 +1,7 @@
 import React from 'react';
 // react router
 import { NavLink } from 'react-router-dom';
+import user from '../image/user.png';
 
 const Header = () => (
   <header className="navbar">
```

Comment:
- [ ] Kindly fix the horizontal scrolling to make your design match the original design.

![image](https://user-images.githubusercontent.com/35535130/172972549-99ca31f5-7684-425a-a01b-ad01afcc40af.png)